README.md

```diff
@@ -62,29 +62,15 @@ kubectl create -f k8s/secret.yaml
 
 1. Clone the repo and navigate to directory
 
-2. Input your application_default_credentials.json information into a secret with the template at k8s/gcr-secret.yaml:
-```bash
-apiVersion: v1
-data:
-  application_default_credentials.json: <base-64 encoded ~/.config/gcloud/application_default_credentials.json>
-kind: Secret
-metadata:
-  name: gcr-secret
-  namespace: kube-system
-  labels:
-    app: gcr-creds
-    version: v1.2
-    kubernetes.io/minikube-addons: gcr-creds
-type: Opaque
-```
+2. Input your application_default_credentials.json information into the secret.yaml template located [here](k8s/secret.yaml#L17):
```

Comment:
On my mac, the application_default_credentials.json` is empy, but my creds are in `credentials`. Is this consistent?